core/variables.js

```diff
@@ -267,11 +268,9 @@ Blockly.Variables.getUses = function(name, workspace) {
 
 /**
  * When a variable is deleted, find and dispose of all uses of it.
- * @param {string} name Name of deleted variable.
- * @param {!Blockly.Workspace} workspace The workspace to delete uses from.
+ * @param {!Array.<!Blockly.Block>} uses An array of blocks using the variable.
  */
-Blockly.Variables.disposeUses = function(name, workspace) {
-  var uses = Blockly.Variables.getUses(name, workspace);
+Blockly.Variables.disposeUses = function(uses) {
```

Comment:
was this public?  If so, we should be careful of changing the API and maybe deprecate the old version in favor of the new one?  I'm not sure what the general practice is for deprecating stuff in blockly... but I have seen the deprecated tag :)